src/pluginscheduler.c

```diff
@@ -62,14 +62,15 @@ plugin_add (plugins_scheduler_t sched, GHashTable *oids_table,
   int category;
   nvti_t *nvti;
   int ret = 0;
+  gchar *tag_value;
+
   if (g_hash_table_lookup (oids_table, oid))
     return 0;
 
   /* Check if the plugin is deprecated */
   nvti = nvticache_get_nvt (oid);
-  if (nvti_tag (nvti)
-      && (g_str_has_prefix (nvti_tag (nvti), "deprecated=1")
-          || strstr (nvti_tag (nvti), "|deprecated=1")))
+  tag_value = nvti_get_tag (nvti, "deprecated");
```

Comment:
Check for null, or apply the suggested change in https://github.com/greenbone/gvm-libs/pull/285#discussion_r338421370

Reply:
gvm-libs is fixed.